organizing_hub/views/call_views.py

```diff
@@ -121,6 +129,48 @@ def can_make_call_for_campaign(user, call_campaign):
     return False
 
 
+def get_or_create_callers(caller_emails):
+    """
+    Get or create callers from a list of caller emails
+
+    Parameters
+    ----------
+    emails : list of str
+        List of email addresses
+
+    Returns
+        -------
+        Callers list
+            Returns list of ints corresponding to callprofile IDs
+    """
+
+    caller_ids = []
+
+    if caller_emails is not None and caller_emails != '':
+        caller_emails = [email.strip() for email in caller_emails.split(",")]
+
+        for email in caller_emails:
+            (user, created) = User.objects.get_or_create(
+                username=email,
```

Comment:
I just checked existing code and every other time the organizing hub looks up a user based on email, it is either `User.objects.get(email__iexact=...` or `User.objects.filter(email__iexact=...`. This means it is case insensitive, and also it is checking the email field. You are doing a case sensitive search, and checking the username field instead of the email field. Is there a reason you are doing it differently? I think probably it should just do it the same way, case insensitive, and email field, right?

Reply:
Agreed about case insensitivity.

But email field is not unique, so we can potentially pull multiple users here if we go with just email. In django username is used as unique to filter on, and since all callers will have email address as username, I elected to query on what we know would be unique instead of what wasn't.

Reply:
while it's possible that django allows multiple user accounts with the same email, our practice is to only have one user account per email. so i think it's fine to either just take the first result or just assume it will only have 1 match max.

not all the usernames in the db are same as email, so it's possible that searching on username for email will miss a user account that does actually have a matching email.

if you want to start supporting multiple user accounts with the same email, that is a longer conversation and i don't think we should do it now. for consistency right now it seems like we should just follow the pattern we already established which is working fine. if you want to change the pattern let's do it later after we can talk it through or research it more?

Reply:
I'll just take first email address here. I think we use get or create for user somewhere else though, does the case insensitivity affect that one too? Should look but not near a computer atm.

Reply:
I don't see us using get or create anywhere for User. And taking a closer look, `filter` is more of an exception, we nearly always do `User.objects.get(email__iexact=...` when looking up a hub user by email.

Reply:
I think I used iexact initially but we wanted to use get or create to cut down on db calls, wonder if you can support case insensitivity with it. Anyways will fix.